services/designation.service.js

```diff
@@ -0,0 +1,32 @@
+const models = require("../models");
+const { sequelize } = require("../models");
+
+const updateDesignation = async (payload) => {
+  const findUser = await models.User.findOne({
+    where: { id: payload.userId },
+  });
+
+  if (!findUser) {
+    throw new Error("user not found");
+  }
+  const checkDesignation = await models.UserDesignationMapping.findOne({
+    where: { user_id: payload.userId, designation_id: payload.designationId },
+  });
+
+  if (checkDesignation) {
+    throw new Error("This relation is already exist");
+  }
+
+  const designation = await models.UserDesignationMapping.update(
+    {
+      designation_id: payload.designationId,
+    },
+    { where: { user_id: payload.userId } }
+  );
+
```

Comment:
don't return payload as it is. use serializer

Reply:
data is already serialized. 

response body -->

 "statusCode": 200,
    "data": {
        "designationId": "a1cb978e-48cf-47d6-9837-b26279f27fa7",
        "userId": "e9c3a24d-8620-49a2-9192-321c981dac32"
    },
    "message": "Success"
}